magento/pwa-studio
# chore: function name 'retreiveAvailableShippingMethods' spelling correction

## Description


Changed the function name `retreiveAvailableShippingMethods` to `retrieveAvailableShippingMethods` on both the occurrences.

## Related Issue
closes #2743 

## Acceptance
### Verification Stakeholders
### Specification
### Verification Steps

1. Go to packages/peregrine/lib/store/actions/checkout/asyncActions.js
2. Check the modified spelling on line 15 and 356.

## Screenshots / Screen Captures (if appropriate)

## Checklist
